src/js/components/icons/IconChevron.js

```diff
@@ -12,25 +12,23 @@ class IconBack extends React.Component {
 
     return (
       <svg
-        height="16px"
```

Comment:
the width and height were being controlled by the parent containers and not these attributes